app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt

```diff
@@ -46,18 +46,18 @@ class UniqueDatabaseManagementTest {
 
     @Test
     fun getDatabaseNamesWorks() = runBlocking {
-        assertThat(uDbMan.getDatabases(), hasItems("demo"))
+        assertThat(uDbMan.getDatabases(), hasItems("test"))
     }
 
     @Test
     fun getDownloadedDatabaseNamesWorks() = runBlocking {
-        uDbMan.downloadDatabase("demo")
-        assertThat(uDbMan.getDownloadedDatabases(), hasItem("demo"))
+        uDbMan.downloadDatabase("test")
+        assertThat(uDbMan.getDownloadedDatabases(), hasItem("test"))
     }
 
     @Test
     fun offlineAndFirestoreDatabasesContainTheSameElements() = runBlocking {
-        val dbName = "demo"
+        val dbName = "test"
```

Comment:
Maybe it would make sense to extract "test" into a `val` to ease future refactorings.